internal/pkg/describe/env.go

```diff
@@ -26,14 +26,14 @@ type EnvDescription struct {
 	Services       []*config.Workload  `json:"services"`
 	Tags           map[string]string   `json:"tags,omitempty"`
 	Resources      []*CfnResource      `json:"resources,omitempty"`
-	EnvironmentVPC EnvironmentVPC      `json:"environment_vpc"`
+	EnvironmentVPC EnvironmentVPC      `json:"environmentVPC"`
 }
 
 // EnvironmentVPC holds the ID of the environment's VPC configuration.
 type EnvironmentVPC struct {
-	ID               string
-	PublicSubnetIDs  []string
-	PrivateSubnetIDs []string
+	ID               string   `json:"id"`
+	PublicSubnetIDs  []string `json:"publicSubnets"`
+	PrivateSubnetIDs []string `json:"privateSubnets"`
```

Comment:
Can we change this for consistency with the `config.ImportVPC` struct? 
```suggestion
	PublicSubnetIDs  []string `json:"publicSubnetIDs"`
	PrivateSubnetIDs []string `json:"privateSubnetIDs"`
```